scripts/2_explore_relationship_AandB_pt2.ipynb

```diff
@@ -0,0 +1,3152 @@
+       "</table>\n",
+       "<p>5 rows × 1100 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "                               PA3423    PA0084    PA2455    PA3029    PA1588  \\\n",
+       "0.1_12hr_CSV86(Pae_G1a).CEL  0.417740  0.417740  0.417740  0.417740  0.417740   \n",
+       "0.1_2hr_CSV86(Pae_G1a).CEL   0.980970  0.980970  0.980970  0.980970  0.980970   \n",
+       "0.1_6hr_CSV86(Pae_G1a).CEL   0.090807  0.090807  0.090807  0.090807  0.090807   \n",
+       "0.1_7hr_CSV86(Pae_G1a).CEL   0.837730  0.837730  0.837730  0.837730  0.837730   \n",
+       "0.1_9hr_CSV86(Pae_G1a).CEL   0.850800  0.850800  0.850800  0.850800  0.850800   \n",
+       "\n",
+       "                               PA2220    PA4656    PA3585    PA1325    PA3067  \\\n",
+       "0.1_12hr_CSV86(Pae_G1a).CEL  0.417740  0.417740  0.417740  0.417740  0.417740   \n",
+       "0.1_2hr_CSV86(Pae_G1a).CEL   0.980970  0.980970  0.980970  0.980970  0.980970   \n",
+       "0.1_6hr_CSV86(Pae_G1a).CEL   0.090807  0.090807  0.090807  0.090807  0.090807   \n",
+       "0.1_7hr_CSV86(Pae_G1a).CEL   0.837730  0.837730  0.837730  0.837730  0.837730   \n",
+       "0.1_9hr_CSV86(Pae_G1a).CEL   0.850800  0.850800  0.850800  0.850800  0.850800   \n",
+       "\n",
+       "                              ...      PA4912    PA2606   PA3887   PA2053  \\\n",
+       "0.1_12hr_CSV86(Pae_G1a).CEL   ...     0.45909  0.480190  0.71269  0.56015   \n",
+       "0.1_2hr_CSV86(Pae_G1a).CEL    ...     0.17085  0.102330  0.28739  0.33152   \n",
+       "0.1_6hr_CSV86(Pae_G1a).CEL    ...     0.50824  0.462670  0.43675  0.58860   \n",
+       "0.1_7hr_CSV86(Pae_G1a).CEL    ...     0.17307  0.087142  0.28269  0.14794   \n",
+       "0.1_9hr_CSV86(Pae_G1a).CEL    ...     0.16421  0.206570  0.22233  0.12298   \n",
+       "\n",
+       "                               PA3396   PA3073   PA4330   PA2133   PA5025  \\\n",
+       "0.1_12hr_CSV86(Pae_G1a).CEL  0.302850  0.81608  0.52592  0.45558  0.00000   \n",
+       "0.1_2hr_CSV86(Pae_G1a).CEL   0.165110  0.32459  0.08787  0.24926  0.27831   \n",
+       "0.1_6hr_CSV86(Pae_G1a).CEL   0.293230  0.57908  0.22536  0.46650  0.19498   \n",
+       "0.1_7hr_CSV86(Pae_G1a).CEL   0.272600  0.39273  0.16383  0.32836  0.14878   \n",
+       "0.1_9hr_CSV86(Pae_G1a).CEL   0.093693  0.40532  0.23176  0.22706  0.21865   \n",
+       "\n",
+       "                              PA3297  \n",
+       "0.1_12hr_CSV86(Pae_G1a).CEL  0.13146  \n",
+       "0.1_2hr_CSV86(Pae_G1a).CEL   0.40542  \n",
+       "0.1_6hr_CSV86(Pae_G1a).CEL   0.66287  \n",
+       "0.1_7hr_CSV86(Pae_G1a).CEL   0.30013  \n",
+       "0.1_9hr_CSV86(Pae_G1a).CEL   0.19401  \n",
+       "\n",
+       "[5 rows x 1100 columns]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Read data\n",
+    "sim_data = pd.read_table(sim_data_file, index_col=0, header=0, compression='xz')\n",
```

Comment:
Just as a heads up if reading in a file you don't need to put `compression='xz'` because pandas will infer the compression extension.

Reply:
That's good to know, thanks